landing_page/app/controllers/microposts_controller.rb

```diff
@@ -44,4 +61,5 @@ def micropost_params
             params[:micropost][:user_id] = current_user.id
             params.require(:micropost).permit(:title, :content, :user_id)
         end
+
```

Comment:
The file is missing a line break at the end. You can tell by the red crossed circle and return arrow next to `end` statement on the next line. This is recommended because that's part of the POSIX standard defines a line. More info on the matter: https://stackoverflow.com/a/729795/1456509